### DISCUSSION TITLE
Is it possible to automatically clear old profiles?

### DISCUSSION THREAD

**Discussion by verasminsk** [NoiseByNorthwest/php-spx#275] [Q&A]
Is it possible to automatically clear old profiles, for example, leave only the N latest profiles?

btw is it possible to filter by the URL mask when the profiler is launched automatically?

**Comment by NoiseByNorthwest** [comment]
> Is it possible to automatically clear old profiles, for example, leave only the N latest profiles?

Currently not, you have to do it yourself in SPX data dir (`/tmp/spx` by default).

> btw is it possible to filter by the URL mask when the profiler is launched automatically?

No, once enabled, every URL rendered by PHP will be profiled within your current browser session.
